lib/models/debugs_bunny/encryption_key.rb

```diff
@@ -4,6 +4,16 @@
 
 require_relative 'concerns/can_generate'
 
+module DaffyLib
+  class EncryptionKey
+    after_rollback :rollback_callback
+
+    def rollback_callback
+      self.class.create(attributes.symbolize_keys)
+    end
+  end
+end
```

Comment:
Arguably, this should be in DaffyLib itself

Reply:
We should just put this in DaffyLib

Reply:
I've added a task in DaffyLib to make this change, and will address it as soon as possible. In the meantime, this needs to go in